include/cpr/status_codes.h

```diff
@@ -0,0 +1,71 @@
+#ifndef _CPR_STATUS_CODES
+#define _CPR_STATUS_CODES
+#include <cstdint>
+namespace cpr {
+namespace status {
+constexpr std::uint32_t HTTP_100_CONTINUE = 100;
```

Comment:
The current Response class uses int32_t for the status code, so this should probably match up.